src/nebula/NebulaAction.h

```diff
@@ -188,9 +188,44 @@ class WalkThroughAction : public core::Action {
     std::string  tag_;
     std::string  col_;
     uint64_t     totalRows_;
+    uint32_t     try_;
+    uint32_t     retryIntervalMs_;
     uint64_t     start_ = 0;
+};
+
+class LookUpAction : public core::Action {
```

Comment:
Maybe LookUpAction could be derived from MetaAction?

Reply:
Actually `LookUpAction` extend `core::Action` and `CreateIndexAction` and `RebuildIndexAction` should extend `MetaAction`